javamesh-package/resources/third-party-license.ftl

```diff
@@ -0,0 +1,108 @@
+<#--
+  #%L
+  Copyright (C) Huawei Technologies Co., Ltd. 2021-2021. All rights reserved
+  %%
+  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
We cannot change the original License to Apache License here, as we don't  own the copyright of the original file.